### DISCUSSION TITLE
How do I get horizontal alignment?

### DISCUSSION THREAD

**Discussion by DKGH** [mantinedev/mantine#6285] [Q&A] [answered]
I have tried Group and Flex with horizontal direction. Neither worked. What could be the problem?

```tsx
import { Group, MantineProvider, Text } from '@mantine/core';

function App() {
  return (
    <MantineProvider>
      <Group>
        <Text variant="default">First</Text>
        <Text variant="default">Second</Text>
        <Text variant="default">Third</Text>
      </Group>
    </MantineProvider>
  )
}

export default App
```

**Comment by rtivital** [comment, marked as answer]
How can I center element on the page?
https://codesandbox.io/s/recursing-galois-t39fbn

How can I center multiple elements on the page?
https://codesandbox.io/s/gracious-water-wvqxm9

How can I center multiple elements horizontally?
https://codesandbox.io/s/busy-http-m74kq5
